autofit/non_linear/nest/dynesty/abstract.py

```diff
@@ -179,6 +179,9 @@ def _fit(self, model: AbstractPriorModel, analysis, log_likelihood_cap=None):
                 config_dict_run = self.config_dict_run
                 config_dict_run.pop("maxcall")
 
+                if not hasattr(sampler, "rstate"):
```

Comment:
Seems pretty funky to attach this function on the fly

Reply:
Its a weird hotfix, I dont totally follow why Dynesty is losing it and cant reproduce it locally but hopefully we'll catch it with the build server somehow.